Solidity/hardhat.config.js

```diff
@@ -2,90 +2,166 @@
  * @type import('hardhat/config').HardhatUserConfig
  */
 
-require('@nomicfoundation/hardhat-toolbox')
-const dotenv = require('dotenv')
-dotenv.config({ path: '../.env' })
+require("@nomicfoundation/hardhat-toolbox");
+require("quai-hardhat-plugin");
```

Comment:
the `quai-hardhat-plugin` is only relevant to the `SolidityX` folder as it downloads the SolidityX compiler. No need to import it in the config here as this only handles vanilla Solidity contracts.